apprise/plugins/__init__.py

```diff
@@ -115,8 +118,11 @@ def __load_matrix(path=abspath(dirname(__file__)), name='apprise.plugins'):
         # Add our module name to our __all__
         __all__.append(plugin_name)
 
-        # Load our module into memory so it's accessible to all
-        globals()[plugin_name] = plugin
+        # Load the module into memory, so it is accessible to all.
+        # TODO: @amotl thinks this completely messes things up, and should
+        #       gradually be removed.
+        if plugin_name not in PLUGINS_WITH_MODERNIZED_TESTS:
+            globals()[plugin_name] = plugin
```

Comment:
The gist of this patch a57f52d is actually that I found that this line, manipulating `globals()`, is apparently only needed for satisfying a few test cases. It is apparently not needed for correct runtime behavior, because the plugin's references will be served via `NOTIFY_MODULE_MAP` anyway. As such, we should gradually phase it out with a subsequent patch.